defi/src/getDBConnection.ts

```diff
@@ -0,0 +1,60 @@
+import postgres from "postgres";
+import setEnvSecrets from "./utils/shared/setEnvSecrets";
+
+let accountsDBConnection: ReturnType<typeof postgres>
+let errorReportsConnection: ReturnType<typeof postgres>
+let coins2Connection: Promise<ReturnType<typeof postgres>>
+
+export function getAccountsDBConnection() {
+  if (!accountsDBConnection)
+    accountsDBConnection = postgres(process.env.ACCOUNTS_DB!);
```

Comment:
unsure if this is an issue but this assumes that this will be available in env (which is not assumed for coins2)